src/hardware/gus.cpp

```diff
@@ -999,11 +999,10 @@ Bitu Gus::ReadFromPort(const Bitu port, const Bitu iolen)
 		return dram_addr < ram.size() ? ram.at(dram_addr) : 0;
 	default:
 #if LOG_GUS
-		LOG_MSG("GUS Read at port 0x%x", port);
+		LOG_MSG("GUS: Read at port 0x%x", static_cast<uint16_t>(port));
```

Comment:
Initially, I wondered why those extra `static_casts` are needed in this commit, but then I noticed the function interfaces. Bleh, Bitu everywhere.

Suggestion: you can write `%#x` instead of `0x%x`.

Reply:
Thanks for the formatting tip; will do.